presto-docs/src/main/sphinx/functions/geospatial.rst

```diff
@@ -131,9 +131,13 @@ Accessors
 
 .. function:: ST_Area(Geometry) -> double
 
-    Returns the area of a polygon using Euclidean measurement on a two dimensional
+    Returns the area of a geometry using Euclidean measurement on a two dimensional
     plane (based on spatial ref) in projected units.
 
+    For Point and LineString types, ST_Area returns 0.0.
```

Comment:
nit: perhaps, drop "ST_Area" for brevity and change "For a GeometryCollection" to "For GeometryCollection type" for consistency 

`For Point and LineString types, returns 0.0.`

`For GeometryCollection type, computes the sum of the areas of the individual geometries.`